### DISCUSSION TITLE
gltf animations not loading correctly

### DISCUSSION THREAD

**Discussion by brbrmensch** [raysan5/raylib#2027] [Q&A] [answered]
had no luck trying to load a model with animations with glb\gltf files with different export configurations from blender, people in discord hinted that it's not finished. is there a hope for it's completion? the example (guy) has .blend and .iqm files, which implies that a conversion is possible, though i had no luck recreating that myself either. guess i have to ask what blender version/converter was it made with?

**Comment by chriscamacho** [comment]
I've had no luck with mixamo.com either (sadly) - the only gltf animation I've had any luck with is the gltf sample cesium guy.  Can't for the life of me work out whats going wrong here!

**Comment by raysan5** [comment, marked as answer]
Unfortunately, glTF animations were not working properly on raylib and support has been removed in latest `raylib 4.0`, maybe it will be added in a future but it would require a better testing before integration.

**Reply by brbrmensch** [reply]
will there be some help regarding .iqm exporting? or is it just me who struggles with it?

**Reply by raysan5** [reply]
I think there is some issue with the Blender exporter, you should ask the creator of the exporter...
